src/js/routes/Ballot/Ballot.jsx

```diff
@@ -162,20 +160,23 @@ export default class Ballot extends Component {
         <h3 className="text-center">{this.emptyMsg()}</h3>
         {emptyBallotButton}
       </div> :
-      <div></div>;
+      null;
+
+    const electionTooltip =  election_date ? <Tooltip id="tooltip">Ballot for {election_date}</Tooltip> : <span></span>;
```

Comment:
Find a better way to create a conditional so that the tooltip doesn't even get triggered if there is no content.